compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1447,8 +1447,11 @@ object Parsers {
       }
     }
 
-    private def makeKindProjectorTypeDef(name: TypeName): TypeDef =
-      TypeDef(name, WildcardTypeBoundsTree()).withFlags(Param)
+    private def makeKindProjectorTypeDef(name: TypeName): TypeDef = {
+      val isVarianceAnnotated = name.lastPart.startsWith("+") || name.lastPart.startsWith("-")
```

Comment:
Couldn't `name.startsWith` be used given that these are user-written names and so should only have one part?

Reply:
@smarter Changed to `name.startsWith`